librz/include/rz_util/rz_base64.h

```diff
@@ -8,9 +8,9 @@ extern "C" {
 #endif
 
 RZ_API size_t rz_base64_encode(char *bout, const ut8 *bin, size_t sz);
-RZ_API int rz_base64_decode(ut8 *bout, const char *bin, int len);
-RZ_API ut8 *rz_base64_decode_dyn(const char *in, int len);
-RZ_API char *rz_base64_encode_dyn(const ut8 *bin, size_t sz);
+RZ_API st64 rz_base64_decode(ut8 *bout, const char *bin, st64 len);
+RZ_API RZ_OWN ut8 *rz_base64_decode_dyn(const char *in, st64 len);
+RZ_API RZ_OWN char *rz_base64_encode_dyn(const ut8 *bin, size_t sz);
```

Comment:
please add also `RZ_NONNULL` to the inputs and for `bout` also `RZ_OUT` (don't forget to update also the .c function definitions).

Reply:
Does `RZ_NONNULL` work together with `rz_return_val_if_fail(src, NULL)`? Because technically, you could pass `NULL`, it's even documented in the docstring.

Reply:
yes, if null is accepted, then you should use `RZ_NULLABLE`